<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupRootDialog</class>
 <widget class="QDialog" name="SetupRootDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup required data files</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/app-icon-large</normaloff>:/app-icon-large</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0">
   <item row="7" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QLabel" name="bigIcon">
     <property name="minimumSize">
      <size>
       <width>128</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="versionLabel">
     <property name="text">
      <string>FlightGear version 3.4.5</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" alignment="Qt::AlignTop">
    <widget class="QPushButton" name="useDefaultsButton">
     <property name="text">
      <string>Use built-in data files</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" alignment="Qt::AlignTop">
    <widget class="QPushButton" name="downloadButton">
     <property name="text">
      <string>Download </string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLabel" name="promptText">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>Replace me</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2" alignment="Qt::AlignTop">
    <widget class="QPushButton" name="browseButton">
     <property name="text">
      <string>Choose folder...</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="useDefaultLabel">
     <property name="text">
      <string>To use the files included with this copy of FlightGear, click this button</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>To download a compressed archive of the files, click the 'Download' button. Once the download is complete, extract the files to a suitable location and choose the folder using the button above.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>To browse to a downloaded copy of the files on your computer, click this button</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
